202305_coastal_ecosystems_TC/impact_calculation/exposure_ecosystem_matching.py

```diff
@@ -0,0 +1,78 @@
+import geopandas as gpd
```

Comment:
You should probably add a header with the license (see other CLIMADA files) and you can add yourself as author.